### ISSUE TITLE
Impossible to navigate the toolbar with the keyboard

### ISSUE THREAD

**Issue by AlbyIanna** [arduino/arduino-ide#1689]
### Describe the problem

The Verify and Upload buttons are not focusable with a keyboard 😢 (it's still possible to use them via shortcuts, but a user should know the shortcuts to use them, so the discoverability it's poor)

### To reproduce

Try to move the focus on the Verify and Upload button (spoiler, you can't 🐛)

### Expected behavior

Try to move the focus on the Verify and Upload button and it works 🐕 

### Arduino IDE version

any

### Operating system

macOS

### Operating system version

any

#### Related

- https://github.com/arduino/arduino-ide/issues/165
- https://github.com/arduino/arduino-ide/issues/360
- https://github.com/arduino/arduino-ide/issues/1561
- https://github.com/arduino/arduino-ide/issues/2839

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the problem still occurs when using the latest [nightly build](https://www.arduino.cc/en/software#nightly-builds)
- [X] My report contains all necessary details
